services/worker-manager/src/api.js

```diff
@@ -340,17 +337,20 @@ builder.declare({
   ].join('\n'),
 }, async function(req, res) {
   const { errorId, workerPoolId } = req.params;
-  const {rows: errors, continuationToken} = await WorkerPoolError.getWorkerPoolErrors(
-    this.db,
-    { errorId, workerPoolId },
-    { query: req.query },
-  );
-  const result = {
-    workerPoolErrors: errors.map(e => e.serializable()),
-    continuationToken,
-  };
+  const {continuationToken, rows} = await paginateResults({
+    query: req.query,
+    fetch: (size, offset) => this.db.fns.get_worker_pool_errors_for_worker_pool(
+      errorId || null,
+      workerPoolId || null,
```

Comment:
Reminder that we can use named arguments :)

Reply:
I was mostly trying to forward-port what we already had.  I'll fix this call.

Reply:
Ah. Sorry about that. I retract my comment.